tests/test_WorkflowFactory.py

```diff
@@ -123,8 +216,14 @@ def test_metagenomic_workflow_creation(self):
         self.assertEqual(wf.protocol_type, PROTOCOL_NAME_ILLUMINA)
         self.assertEqual(wf.assay_type, ASSAY_NAME_METAGENOMIC)
 
-        with self.assertRaisesRegex(
-                PipelineError, 'There are no fastq files for FastQCJob'):
+        self._inject_data(wf)
+
+        # Metagenomic is a valid data type in the default qiita test
+        # database but job-id: 78901 doesn't exist; however, if we get
+        # to here, it means that all steps have ran to completion
+        # and the system is trying to create the preps.
+        with self.assertRaisesRegex(RuntimeError, 'invalid input '
+                                    'syntax for type uuid: "78901"'):
```

Comment:
I am confused by the error message and the comment here.  What does "invalid input syntax for type uuid <whatever>" *mean*?  The comment mentions that 78901 is a job id, so why does the error message refer to it as a "type uuid"?

Reply:
Qiita job ids are UUID in the database, this tests uses 78901 as the job id so the database complains about the format. IMOO it is fine to let it fail here as is the last step of the processing.

Reply:
Aha, got it!  For the benefit of non-experts like me reading the tests, would you consider having these repeated comments say essentially what you just did? 

```
        # Metagenomic is a valid data type in the default qiita test
        # database but the test job-id, 78901, isn't in the 
        # format of a uuid as expected; however, if we get
        # to here, it means that all steps have run to completion
```